### DISCUSSION TITLE
Patching Mutter paint related artifacting

### DISCUSSION THREAD

**Discussion by hanaral** [elementary/gala#1071] [Ideas]
Since #605 doesn't seem to have anyone offering to fix it, what is the average user going to do? I personally get distracted from this issue greatly and cannot imagine how bad it is if you are sensitive to flickering.
The issue evidently hasn't fixed itself over the last 2.5 years it has existed, and the only proposed fix is by [to patch it](https://github.com/elementary/gala/issues/605#issuecomment-780123742). I have yet to find a single peice of hardware that doesn't exhibit this behaviour.

I hope that this project remains as a stable release model and patches upstream problems rather than pointing users to complain elsewhere.
@danrabbit sorry to add another notification to your list

**Comment by hanaral** [comment]
It feels cheap, but it's already pretty obvious that this needs to get patched at this point. 
@cassidyjames Could you please start some internal dialogue to get it packaged on Odin and maybe even backported to 5.2?
I'd use a patched version but it has lots of deps that aren't available on elementary yet. (even though the patch is literally just commenting out a single line)

**Comment by lenemter** [comment]
Closing as resolved
